daemon/k8s_watcher.go

```diff
@@ -1185,14 +1182,22 @@ func (d *Daemon) addCiliumNetworkPolicyV2(ciliumV2Store cache.Store, cnp *cilium
 	k8sCM.UpdateController(ctrlName,
 		controller.ControllerParams{
 			DoFunc: func() error {
-				return cnpNodeStatusController(ciliumV2Store, cnp, rev, scopedLog, policyImportErr)
+				return cnpNodeStatusController(ciliumNPClient, ciliumV2Store, cnp, rev, scopedLog, policyImportErr, node.GetName())
 			},
 		},
 	)
 	return policyImportErr
 }
 
-func cnpNodeStatusController(ciliumV2Store cache.Store, cnp *cilium_v2.CiliumNetworkPolicy, rev uint64, logger *logrus.Entry, policyImportErr error) error {
+func cnpNodeStatusController(
```

Comment:
We should try to avoid formatting changes in feature and bug commits.

Reply:
it's not formatting, lots of new fields were added.